docs/src/css/custom.css

```
@@ -650,100 +627,237 @@ a:hover {
+
+/* Homepage */
+.HomePage {
+  width: 100%;
+  overflow-x: hidden;
+}
+
+/* Section */
+.Section {
+  width: 100%;
+  padding-top: 50px;
+  padding: 0 1.25rem;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-top



(lint/suspicious/noShorthandPropertyOverrides)